modules/PyTestControlModule/manifest.yaml

```diff
@@ -0,0 +1,24 @@
+description: >-
+  This module provides the control interface to automatically run CI/CD tests on everest-core
+config:
+  device:
+    description: >-
+      change this to add control commands
+    type: string
+    default: abcd
+provides:
+  main:
+    interface: empty
+    description: This module provides no outward-facing interface
+requires:
+  test_control:
+    interface: car_simulator
+  connector_1:
+    interface: evse_manager
+    min_connections: 1
```

Comment:
min_connections should always be 0 in order to be able to execute tests with configs that do not load the module.

Reply:
Changed min_connections and added debug messages on affected functions.